lib/schema_plus/foreign_keys/middleware/sql.rb

```diff
@@ -2,22 +2,23 @@ module SchemaPlus::ForeignKeys
   module Middleware
     module Sql
       module Table
-        def after(env)
-          foreign_keys = if env.table_definition.foreign_keys.is_a? Array
-                           env.table_definition.foreign_keys
-                         else
-                           env.table_definition.foreign_keys.values.tap { |v| v.flatten! }
-                         end
+        if Gem::Requirement.new('< 5.0.0.alpha.1').satisfied_by?(::ActiveRecord.version)
```

Comment:
WBN for schema_monkey to provide some support for version-specific code.  I'm not sure how best to do it.  A simple thing could be to provide a helper to be able to do this test with less verbose boilerplate.   But just thinking aloud, this is fine for now.